installer/release.sh

```diff
@@ -132,7 +132,7 @@ fi
 
 printf "; List of files\n%s\n%s\n%s\n%s\n%s\n" \
 	"Source: \"{#SourcePath}\\package-versions.txt\"; DestDir: {app}\\etc; Flags: replacesameversion; AfterInstall: DeleteFromVirtualStore" \
-	"Source: \"{#SourcePath}\\usr\\share\\git\\ReleaseNotes.css\"; DestDir: {app}\\usr\\share\\git; Flags: replacesameversion; AfterInstall: DeleteFromVirtualStore" \
+	"Source: \"{#SourcePath}\\..\\usr\\share\\git\\ReleaseNotes.css\"; DestDir: {app}\\usr\\share\\git; Flags: replacesameversion; AfterInstall: DeleteFromVirtualStore" \
```

Comment:
If we already move the `.css` out of the `installer/`  directory, we might just as well get rid of the directory structure. The intention of keeping the `.css` in the subdirectory was to be able to preview the release notes locally. However, since `installer/release.sh` learned about the `--render-release-notes` option, it might just as well render them *slightly* differently, to pick up the `.css` directly from the `build-extra/` directory.

As this is a clean-up of my mess, I'll gladly do it myself after merging your work.